src/main/scala/com/gu/subscriptions/cas/monitoring/CloudWatch.scala

```diff
@@ -23,11 +24,19 @@ trait CloudWatch extends LazyLogging  {
     client
   }
 
-  trait LoggingAsyncHandler extends AsyncHandler[PutMetricDataRequest, Void]
-  {
+  trait LoggingAsyncHandler extends AsyncHandler[PutMetricDataRequest, Void] {
+
+    val rateLimiter = RateLimiter.create(0.1)
```

Comment:
This should be fine, so long as the subclass of LoggingAsyncHandler effectively ends up being at least a near-singleton?

We had an awful issue a while back where a similar assumption was not true: https://github.com/guardian/membership-attribute-service/pull/82